### PR TITLE
Pistol changes

### PR DESCRIPTION
**Pull request by AmoryBlaine**
- Marine vendors loses pistol ammo
- Marine vendors loses non-standard pistols
- Marine service pistol magazines added to gear lockup
- VP78 removed until redesigned/rebalance
- Various desc changes
- Various pistol audio changes
- Marine vendor no longer carries ammo
- Marine prep zone has service pistol magazines
- MK-45 Highpower Automag is now called the HG-45 Super and is added to loadout
- HG-45 Super and 1911 now share same ammo, dealing 45 damage (4 rounds to crit a drone)
- 1911 now has a silver HG-45 magazine instead of legacy
- HG-45 black now looks like a USCM pistol sharing palette with the M41A
- Pistols do not trigger beeping when auto ejecting magazines
- Black Spearhead added to the loadout

### CODE REVIEW

File: code/modules/projectiles/guns/revolvers.dm [hunk 1]
```diff
@@ -700,8 +700,8 @@
 //MARSHALS REVOLVER //Spearhead exists in Alien cannon.
 
 /obj/item/weapon/gun/revolver/cmb
-	name = "\improper CMB Spearhead autorevolver"
-	desc = "An automatic revolver chambered in .357 commonly issued to Colonial Marshals."
+	name = "\improper Spearhead Armoury autorevolver"
+	desc = "A semi-automatic revolver chambered in .357 commonly issued to Colonial Marshals."
```
Comment (mesnomer) [inline code/modules/projectiles/guns/revolvers.dm:704]: Honestly I'd change the name from "... Armoury autorevolver" to "... Armoury revolver."

Whether semi-automatic is accurate is debatable here, but "autorevolver" is just straight up inaccurate as it's a conflation with how the Mateba operates. While the Mateba did supposedly inspire the Spearhead and DOES have a reciprocating element (ie the whole dang gun from the grip up) to re-cock the hammer with every shot for full SA usage, the Spearhead lacks that element.

Comment (AmoryBlaine) [reply code/modules/projectiles/guns/revolvers.dm:704]: Sure, why not

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Doubleumc — APPROVED** [review]

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: code/modules/projectiles/guns/pistols.dm [hunk 2]
```diff
@@ -102,37 +104,89 @@
 	damage_mult = BASE_BULLET_DAMAGE_MULT
 
 
+//VP70 - Counterpart to M1911, offers burst and capacity ine exchange of low accuracy and damage.
+
+/obj/item/weapon/gun/pistol/vp70
+	name = "\improper VP70 M5 service pistol"
+	desc = "Standard issue semi-automatic USCM service pistol. Recently replacing the M4A3, it retains its predecessor's 9mm chambering but offers both a higher magazine capacity and a 3-round burst selector."
+	icon = 'icons/obj/items/weapons/guns/guns_by_faction/uscm.dmi'
+	icon_state = "vp70"
```
Comment (Doubleumc) [inline code/modules/projectiles/guns/pistols.dm:113]: There's no `vp70` in the `'icons/obj/items/weapons/guns/guns_by_faction/uscm.dmi'` file.

File: code/game/objects/items/storage/misc.dm [hunk 3]
```diff
@@ -111,48 +111,117 @@
...
-	new /obj/item/weapon/gun/pistol/m1911(src)
-	new /obj/item/ammo_magazine/pistol/m1911(src)
-	new /obj/item/ammo_magazine/pistol/m1911(src)
+/obj/item/storage/box/loadout/clf/fill_preset_inventory()
+	handle_item_insertion(new /obj/item/weapon/gun/pistol/clfpistol())
+	for(var/i = 1 to storage_slots - 1)
+		new /obj/item/ammo_magazine/pistol/clfpistol(src)
 
-/obj/item/storage/box/co2_knife
+/obj/item/storage/box/loadout/co2_knife
```
Comment (Doubleumc) [inline code/game/objects/items/storage/misc.dm:224]: Will need to repath `co2_knife` in all the maps as well:
image

**Review by Doubleumc — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/pistols.dm (hunk 2 above)
Comment (AmoryBlaine) [reply code/modules/projectiles/guns/pistols.dm:113]: Was deleted in icon merge, fixed now

File: code/game/objects/items/storage/misc.dm (hunk 3 above)
Comment (AmoryBlaine) [reply code/game/objects/items/storage/misc.dm:224]: Fixed

**Review by Doubleumc — APPROVED** [review]

**Pull request merged**